quickwit-index/src/actors/uploader.rs

```diff
@@ -146,14 +126,14 @@ async fn put_split_files_to_storage(
         .await?;
 
     let elapsed_secs = start.elapsed().as_secs();
-    let file_statistics = manifest.file_statistics();
+    let file_statistics = split.file_statistics.clone();
     info!(
         min_file_size_in_bytes = %file_statistics.min_file_size_in_bytes,
         max_file_size_in_bytes = %file_statistics.max_file_size_in_bytes,
         avg_file_size_in_bytes = %file_statistics.avg_file_size_in_bytes,
-        split_size_in_megabytes = %manifest.split_size_in_bytes / 1000,
+        split_size_in_megabytes = %split.bundle_offsets.bundle_file_size / 1000,
         elapsed_secs = %elapsed_secs,
-        throughput_mb_s = %manifest.split_size_in_bytes / 1000 / elapsed_secs.max(1),
+        throughput_mb_s = %split.bundle_offsets.bundle_file_size as u64  / 1000 / elapsed_secs.max(1),
```

Comment:
can we put the size_in_mb in a variable an compute it once instead of twice?

Reply:
yep